lldb/test/Shell/SymbolFile/DWARF/x86/simple-template-names-context.cpp

```diff
@@ -4,8 +4,8 @@
 
 // REQUIRES: lld
 
-// RUN: %clang --target=x86_64-pc-linux -c %s -o %t-a.o -g -gsimple-template-names -DFILE_A
-// RUN: %clang --target=x86_64-pc-linux -c %s -o %t-b.o -g -gsimple-template-names -DFILE_B
```

Comment:
We should keep the existing test, but you can add another set of RUN lines to it.
If you do that, it would also be good to rename the test to reflect its wider scope. Maybe to something about type definition searching, as that's the common theme in the two cases.

Reply:
Added another set of RUN for -fdebug-types-section